client/state/posts/test/reducer.js

```diff
@@ -205,7 +238,7 @@ describe( 'reducer', () => {
 			} );
 		} );
 
-		it( 'never persists state because this is not implemented', () => {
+		it( 'never persists state', () => {
```

Comment:
Oh, or is it that these are things that should not be persisted (rather than just haven't been implemented yet), so the previous language wasn't quite accurate.